lib/generic/modules/metabolic_syndrome_disease.json

```diff
@@ -0,0 +1,651 @@
+      "attribute" : "diabetic_kidney_damage",
+      "value" : 0,
+      "direct_transition" : "Initial_Eye_Health"
+    },
+
+    "Initial_Eye_Health" : {
+      "type" : "SetAttribute",
+      "attribute" : "diabetic_eye_damage",
+      "value" : 0,
+      "direct_transition" : "Initial_Nerve_Health"
+    },
+
+    "Initial_Nerve_Health" : {
+      "type" : "SetAttribute",
+      "attribute" : "diabetic_nerve_damage",
+      "value" : 0,
+      "direct_transition" : "Age_and_BG_Guard"
+    },
+
+    "Age_and_BG_Guard" : {
+      "type" : "Guard",
+      "allow" : {
+      "condition_type": "And",
+      "conditions": [
+        {
+          "condition_type": "Age",
+          "operator": ">=",
+          "quantity": 16,
+          "unit": "years"
+        },
+        {
+          "condition_type" : "Vital Sign",
+          "vital_sign" : "blood_glucose",
+          "operator" : "is not nil"
+        }
+      ]
+      },
+      "direct_transition" : "Potential_Diabetes"
+    },
+
+    "Potential_Diabetes" : {
+      "type" : "Delay",
+      "exact" : { "quantity" : 7, "unit" : "days" },
+      "conditional_transition" : [
+        {
+          "condition" : {
+            "condition_type" : "And",
+            "conditions" : [
+              {
+                "condition_type" : "Attribute",
+                "attribute" : "hypertension",
+                "operator" : "is nil"
+              },
+              {
+                "condition_type": "Age",
+                "operator": ">=",
+                "quantity": 18,
+                "unit": "years"
+              }
+            ]
+          },
+          "transition" : "Chance_to_Onset_Hypertension"
+        },
+        {
+          "condition" : {
+            "condition_type" : "Vital Sign",
+            "vital_sign" : "blood_glucose",
+            "operator" : ">",
+            "value" : 6.5
+          },
+          "transition" : "Onset_Diabetes"
+        },
+        {
+          "condition" : {
+            "condition_type" : "Vital Sign",
+            "vital_sign" : "blood_glucose",
+            "operator" : ">",
+            "value" : 5.7
+          },
+          "transition" : "Onset_Prediabetes"
+        },
+        {
+          "transition" : "Potential_Diabetes"
+        }
+      ]
+    },
+
+    "Chance_to_Onset_Hypertension" : {
+      "type" : "Simple",
+      "distributed_transition" : [
+        { "distribution" : 0.594, "transition" : "Onset_Hypertension" },
+        { "distribution" : 0.406, "transition" : "No_Hypertension" }
+      ],
+      "remarks" : "probability_given_diabetes: 0.594 #  (1.0==100%) http://www.cdc.gov/MMWr/preview/mmwrhtml/su6203a24.htm#Tab"
+    },
+
+    "Onset_Hypertension" : {
+      "type" : "SetAttribute",
+      "attribute" : "hypertension",
+      "value" : true,
+      "direct_transition" : "Potential_Diabetes"
+    },
+
+    "No_Hypertension" : {
+      "type" : "SetAttribute",
+      "attribute" : "hypertension",
+      "value" : false,
```

Comment:
Nah, we should use the booleans. If you have correctly identified the issue, then I think this means we should fix the Graphviz renderer.